forge-gui/res/cardsfolder/c/case_of_the_locked_hothouse.txt

```diff
@@ -5,6 +5,7 @@ S:Mode$ Continuous | Affected$ You | AdjustLandPlays$ 1 | Description$ You may p
 T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | CheckSVar$ X | SVarCompare$ GE7 | IsPresent$ Card.Self+IsUnsolved | TriggerZones$ Battlefield | Execute$ TrigSolve | TriggerDescription$ To solve — You control seven or more lands. (If unsolved, solve at the beginning of your end step.)
 SVar:TrigSolve:DB$ AlterAttribute | Defined$ Self | Attributes$ Solved
 SVar:X:Count$Valid Land.YouCtrl
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | AffectedZone$ Library | MayLookAt$ You | IsPresent$ Card.Self+IsSolved | Description$ You may look at the top card of your library any time.
```

Comment:
helpers should still be marked Secondary to avoid duplicated text (sorry boss)

![grafik](https://github.com/Card-Forge/forge/assets/8506892/ca824b4c-4807-46df-8fcf-6452ccd77a25)

Reply:
Haha no worries. I just saw this while trying to do my other PR and wanted to fire it off fast.